apps/aurora-portal/src/client/Identity/Auth/Nav/Nav.tsx

```diff
@@ -0,0 +1,38 @@
+import { useLocation } from "wouter"
+// @ts-expect-error missing types
+import { Button, Icon } from "@cloudoperators/juno-ui-components"
+import { trpc } from "../../../trpcClient"
+
+export function Nav() {
+  const checkLoginStatus = trpc.identity.getAuthStatus.useQuery()
+  const logoutMutation = trpc.identity.logout.useMutation()
+  const setLocation = useLocation()[1]
+
+  const login = () => {
+    setLocation("/auth/signin")
+  }
+
+  const logout = () => {
+    logoutMutation.mutate()
+  }
+
+  if (logoutMutation.isSuccess) checkLoginStatus.refetch()
```

Comment:
Not sure here if it makes sense... But this would be the pattern reactQuery works. Makes sense?

```js
const logout = () => {
  logoutMutation.mutate(undefined, {
    onSuccess: () => {
      checkLoginStatus.refetch();
    },
  });
};
```

Reply:
cool, I didn't know it. You can see who is the react-query expert ;)